clang-tools-extra/clang-doc/JSONGenerator.cpp

```diff
@@ -84,8 +84,24 @@ serializeLocation(const Location &Loc,
   return LocationObj;
 }
 
+/// Insert comments into a key in the Description object.
+///
+/// \param Comment Either an Object or Array, depending on the comment type
+/// \param Key     The type (Brief, Code, etc.) of comment to be inserted
 static void insertComment(Object &Description, json::Value &Comment,
                           StringRef Key) {
+  // The comment has a Children array for the actual text, with meta attributes
+  // alongside it in the Object.
+  if (auto *Obj = Comment.getAsObject(); Obj) {
```

Comment:
```suggestion
  if (auto *Obj = Comment.getAsObject()) {
```
Isn't this equivalent? Am I missing something?